operator/v1/types_console.go

```diff
@@ -213,16 +254,16 @@ type AddPage struct {
 type PerspectiveState string
 
 const (
-	Enabled      PerspectiveState = "Enabled"
-	Disabled     PerspectiveState = "Disabled"
-	AccessReview PerspectiveState = "AccessReview"
+	PerspectiveEnabled      PerspectiveState = "Enabled"
+	PerspectiveDisabled     PerspectiveState = "Disabled"
+	PerspectiveAccessReview PerspectiveState = "AccessReview"
 )
 
-// PerspectiveAccessReview defines the visibility of the perspective depending on the access review checks.
+// ResourceAttributesAccessReview defines the visibility of the perspective depending on the access review checks.
```

Comment:
Nit: A generic type name `ResourceAttributesAccessReview` with a really specific job "defines the visibility of the perspective", `PerspectiveVisibilityAccessReview` might be an option - or updating the doc.